backend/build.gradle.kts

```diff
@@ -23,6 +23,9 @@ dependencies {
     compileOnly("org.projectlombok:lombok:1.18.20")
     annotationProcessor("org.projectlombok:lombok:1.18.20")
     implementation("org.projectlombok:lombok:1.18.28")
+    implementation("org.bitbucket.b_c:jose4j:0.6.0")
```

Comment:
why do we need bitbucket?

Reply:
Thats the source for the library, its not bitbucket but jose4j, which is a library for handling JWTs